portal-impl/test/unit/com/liferay/portal/zip/ZipWriterImplTest.java

```diff
@@ -305,6 +305,29 @@ public void testConstructor() {
 		file2.delete();
 	}
 
+	@Test
+	public void testConstructorWithFilePathContainSpecialCharacter()
+		throws IOException {
+
+		Path tempZipFilePath = Files.createTempFile(
+			Paths.get(SystemProperties.get(SystemProperties.TMP_DIR)), null,
```

Comment:
@JanisZhang I can see this is in the `setUp()` method, but it's redundant here. `SystemProperties.TMP_DIR` is `java.io.tmpdir`, which is by default set by the JDK. When you use the `Files.createTempFile` API, you don't actually need to provide the path to the temp dir, as it will use the filesystem's default temp dir.